web_pivot_computed_measure/README.rst

```diff
@@ -0,0 +1,89 @@
+==========================
+Web Pivot Computed Measure
+==========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/12.0/web_pivot_computed_measure
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_pivot_computed_measure
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Adds support for computed measures on the pivot view
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Go to pivot view and click on the "Measures" menu, you will see
+a new option called 'Computed Measure'.
```

Comment:
Add a bit more description of the usage (select first measure, second, the operation to be done, etc).

What about the developer option to put the formula?

Reply:
Add also:

- that you can use a computed measure as operand for other computed measures.
- explain that involved measures are added in the pivot automatically and they are needed for proper computed measure calculus, so if one of them is removed, the computed measure will be removed from the pivot table as well. This requirement is real, isn't it?

Reply:
You can't remove "measures" that are being used in a "computed measure". Need disable the computed measure first.

Reply:
Then please comment this in the README